### PR TITLE
Feature welcome

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding controller and view for greeting
- Adding developers tools's dependency
- I considered that this PR can contain two trello cards due to the first card's requirement
## Trello Card
https://trello.com/c/aWk3psyE
https://trello.com/c/cxNodQ5J
## Test
image

### CODE REVIEW

File: src/main/java/com/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@Controller
+public class BookController {
+
+    @GetMapping("/greeting")
+    public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
+            Model model) {
+        model.addAttribute("name", name);
+        return "greeting";
+    }
```
Comment (torsello) [inline src/main/java/com/wolox/training/controllers/BookController.java:19]: Hi @JoelIvanStella, what do you think about adding constants to these strings?. It's very useful to reuse them and make it easier to modify them :wink:

**Review by torsello — CHANGES_REQUESTED** [review]

File: src/main/java/com/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/controllers/BookController.java:19]: Yes Mati, it`s a good practice. I push the change ;)

**Review by jdhiguita — APPROVED** [review]
Great job!

**Comment by torsello** [comment]
LGTM!

**Review by torsello — APPROVED** [review]

**Pull request merged**